toptal/picasso
# fix(Select): make Select a generic

### Description

Make all `Select` components generic over `value`, `options`, and `multiple` properties types.

### How to test

- Open https://github.com/toptal/picasso/blob/generic-select/packages/picasso/src/Select/story/Autofill.example.tsx
- Change `value`'s type to `number`
- _Expected:_ Compiler error: it should request a `string`
- Add `multiple` property
- _Expected:_ Compiler error: it should request `string[]` in `value` and event handler

### Review
